byron-proxy/src/exec/Main.hs

```diff
@@ -475,8 +477,11 @@ main = do
             , indexFilePath = bpoIndexPath bpo
             , slotsPerEpoch = epochSlots
             }
-      withDB dbc $ \db -> do
-        let server = runServer (Logging.convertTrace trace) (bpoServerOptions bpo) epochSlots db
-            client = runClient (Logging.convertTrace trace) (bpoClientOptions bpo) genesisConfig epochSlots db
+          -- Trace DB writes in such a way that they appear in EKG.
+          dbTracer = flip contramap (Logging.convertTrace trace) $ \(DB.DBWrite (SlotNo count)) ->
+            ("db", Monitoring.Info, Monitoring.LogValue "block count" (Monitoring.PureI (fromIntegral count)))
```

Comment:
@CodiePP Is this enough to get an EKG gauge?

Reply:
Yes.